[BUG] test_tar_distribution fails when running http_logs as the assertions fail

`http_logs` has assertions that its queries return the expected number of results. These assertions fail when running in test mode:
```
opensearch-benchmark execute_test --distribution-version="1.0.0" --workload="http_logs" --test-mode --provision-config-instance=4gheap --target-hosts=127.0.0.1:19200 --kill-running-processes --on-error='abort' --enable-assertions

   ____                  _____                      __       ____                  __                         __
  / __ \____  ___  ____ / ___/___  ____ ___________/ /_     / __ )___  ____  _____/ /_  ____ ___  ____ ______/ /__
 / / / / __ \/ _ \/ __ \\__ \/ _ \/ __ `/ ___/ ___/ __ \   / __  / _ \/ __ \/ ___/ __ \/ __ `__ \/ __ `/ ___/ //_/
/ /_/ / /_/ /  __/ / / /__/ /  __/ /_/ / /  / /__/ / / /  / /_/ /  __/ / / / /__/ / / / / / / / / /_/ / /  / ,<
\____/ .___/\___/_/ /_/____/\___/\__,_/_/   \___/_/ /_/  /_____/\___/_/ /_/\___/_/ /_/_/ /_/ /_/\__,_/_/  /_/|_|
    /_/

[INFO] Preparing for test execution ...
[INFO] Executing test with workload [http_logs], test_procedure [append-no-conflicts] and provision_config_instance ['4gheap'] with version [1.0.0].

Running delete-index                                                           [100% done]
Running create-index                                                           [100% done]
Running check-cluster-health                                                   [100% done]
Running index-append                                                           [100% done]
Running refresh-after-index                                                    [100% done]
Running force-merge                                                            [100% done]
Running refresh-after-force-merge                                              [100% done]
Running wait-until-merges-finish                                               [100% done]
Running default                                                                [100% done]

Running term                                                                   [  0% done][ERROR] Cannot execute_test. Error in load generator [0]
	Cannot run task [term]: Expected [hits] to be == [10000] but was [151].
```

The integration tests are run with the parameters `--on-error='abort' --enable-assertions` but also in test mode, causing the above error to be thrown

--- a/it/__init__.py
+++ b/it/__init__.py
@@ -94,7 +94,7 @@ def execute_test(cfg, command_line):
     This method should be used for benchmark invocations of the test_execution command.
     It sets up some defaults for how the integration tests expect to run test_executions.
     """
-    return osbenchmark(cfg, f"execute_test {command_line} --kill-running-processes --on-error='abort' --enable-assertions")
+    return osbenchmark(cfg, f"execute_test {command_line} --kill-running-processes --on-error='abort'")
 
 
 def shell_cmd(command_line):

